IdentityCore/src/controllers/MSIDSilentController.m

```diff
@@ -96,7 +96,7 @@ - (void)acquireToken:(nonnull MSIDRequestCompletionBlock)completionBlock
     
     __auto_type request = [self.tokenRequestProvider silentTokenRequestWithParameters:self.requestParameters
                                                                          forceRefresh:self.forceRefresh];
-    
+    request.skipLocalRt = self.requestParameters.canPerformSilentSsoExtRequest;
```

Comment:
Would it be perhaps more straightforward for it to be a parameter on the controller itself?